src/Appwrite/Utopia/Response/Model/Migration.php

```diff
@@ -34,12 +34,6 @@ public function __construct()
                 'default' => '',
                 'example' => 'pending',
             ])
-            ->addRule('stage', [
```

Comment:
whats the difference between stage and status? why is this removed? 

This needs to be released as a patch version so we should not have breaking changes. If it has to be removed, then we need a response filter for it

Reply:
I've added a response filter, please re-review